x/data/server/features/register_resolver.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Register Resolver
+
+  Scenario: data is registered to the resolver when the data has been anchored
+    Given a valid content hash
```

Comment:
this can be in the `Background` section and let's use a real example

Reply:
Moved to background and updated to use real example with doc string.